BookingsApi/BookingsApi.DAL/Commands/UpdateJudiciaryParticipantCommand.cs

```diff
@@ -0,0 +1,49 @@
+namespace BookingsApi.DAL.Commands
+{
+    public class UpdatedJudiciaryParticipant
+    {
+        public string DisplayName { get; set; }
+        public string PersonalCode { get; set; }
+        public JudiciaryParticipantHearingRoleCode HearingRoleCode { get; set; }
+    }
+    
+    public class UpdateJudiciaryParticipantCommand : ICommand
+    {
+        public Guid HearingId { get; set; }
+        public UpdatedJudiciaryParticipant UpdatedJudiciaryParticipant { get; set; }
+
+        public UpdateJudiciaryParticipantCommand(Guid hearingId, UpdatedJudiciaryParticipant updatedJudiciaryParticipant)
+        {
+            HearingId = hearingId;
+            UpdatedJudiciaryParticipant = updatedJudiciaryParticipant;
+        }
+    }
+
+    public class UpdateJudiciaryParticipantCommandHandler : ICommandHandler<UpdateJudiciaryParticipantCommand>
+    {
+        private readonly BookingsDbContext _context;
+        
+        public UpdateJudiciaryParticipantCommandHandler(BookingsDbContext context)
+        {
+            _context = context;
+        }
+        
+        public async Task Handle(UpdateJudiciaryParticipantCommand command)
```

Comment:
if the role can change, then include the participants. We shouldn't be able to update a JoH's role to judge if there's already a judge added

Reply:
this is currently handled in the Hearing model. So it will throw an exception when trying to update the role to judge if another judge already exists:

```
if (newHearingRoleCode == JudiciaryParticipantHearingRoleCode.Judge && DoesJudgeExist(personalCodeToIgnore: personalCode))
            {
                throw new DomainRuleException(nameof(personalCode), DomainRuleErrorMessages.ParticipantWithJudgeRoleAlreadyExists);
            }
```

Reply:
Sorry misread the command!